src/basic/locale-util.c

```diff
@@ -357,6 +357,7 @@ const char *special_glyph(SpecialGlyph code) {
                         [SPECIAL_GLYPH_SLIGHTLY_UNHAPPY_SMILEY] = ":-(",
                         [SPECIAL_GLYPH_UNHAPPY_SMILEY]          = ":-{",
                         [SPECIAL_GLYPH_DEPRESSED_SMILEY]        = ":-[",
+                        [SPECIAL_GLYPH_LOCK_AND_KEY]            = "o-,"
```

Comment:
Shouldn't this be `o~` ?

edit: Answering myself: since this isn't actually used, it totally doesn't matter ;)

Reply:
hmm, yeah, the o~ thing looks good too. i guess we can rethink this if we ever make use of the emoji in ascii ;-)